RELEASENOTES-1.4.docu

```diff
@@ -537,4 +537,29 @@
       template now allows for its implementations of <tt>unmapped_read</tt>,
       <tt>unmapped_write</tt>, and <tt>unmapped_get</tt> to be
       overridden. </add-note></build-id>
+  <build-id _6="next" _7="next"><add-note> It's now allowed to instantiate
+      multiple unrelated templates that offer different <tt>shared</tt>
+      implementations of the same method, as long as that method is declared
+      in a common ancestor template, and the conflict is resolved through
+      an implementation that overrides all the unrelated implementations
+      <bug number="SIMICS-20431"/>. This mirrors the pre-existing semantics for
+      non-<tt>shared</tt> methods.</add-note></build-id>
+  <build-id _6="next" _7="next"><add-note> Added the `templates` member for
+      all objects and values of template types, which allows for making calls
+      to any particular implementation of a method as provided by a specified
+      template <bug number="SIMICS-22264"/>. Example syntax:
+      <pre>
+      // Call the `write_register()` implementation as provided by the
+      // `register` template on the `r` register of the bank `b`.
+      b.r.templates.register.write_register(...);
+      </pre>
+      Calls of this form are called <em>Template-Qualified Method Implementation
+      Calls</em>. This feature is primarily offered to make it more ergonomic to
+      write overriding methods that get defined in order to resolve conflicts
+      introduced from the usage of multiple unrelated templates that provide
+      differing implementations of the same methods. `templates` provides a
+      means for such an override to reference the implementations it overrides,
+      whereas it previously couldn't (as <tt>default()</tt> would be ambiguous.)
+      For more information, see the DML 1.4 Reference
+      Manual.</add-note></build-id>
```

Comment:
Verbose. I could leave it at the first paragraph, but that would mean we don't say anything about intended usage here.

Reply:
After the example, you might just wrap up with something like "this is sometimes useful to resolve conflicts in multiple inheritance"

Reply:
How about...
> Calls of this form are called <em>Template-Qualified Method Implementation Calls</em>, and are primarily meant to make it easier to resolve conflicts in multiple inheritance: it allows any method that overrides multiple implementations from unrelated templates to individually reference and call the implementations it overrides.

Reply:
Yes, that's better.